service/apis/qsl_service.go

```diff
@@ -81,11 +81,11 @@ func NewQSLService(host db.IDGClient) *QSLService {
 // -> , $name: string, $k8sobj: string, $resourceid: string
 // pagination
 // $$limit=2,offset=2
-// -> limit: 2,offset: 2
-func CreateFiltersQuery(filterlist string) (string, string, string, error) {
+// -> first: 2,offset: 2
```

Comment:
should be `limit` or?

Reply:
it used to map qsl to dgraph query, so it should be `first`